### DISCUSSION TITLE
My account suspended

### DISCUSSION THREAD

**Discussion by superdened** [community/community#144647] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello.

 My account https://github.com/denys-khvashchenko was blocked 2 weeks ago.  I didn't receive a blocking email. 
 I made a recovery request and received Ticket ID: 3082065, but after almost 2 weeks I have not received any response.

 How long do users usually wait for an answer to such questions?

**Comment by davevad93** [comment]
Hi @superdened ,

Support can take a while to respond because they have a lot of daily requests to deal with, it can take even weeks so be patient. I hope you'll get a reply as soon as possible and solve your problem.

**Comment by mecodeatlas** [comment]
Hi @superdened!

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
